R/generate_nowcasts.R

```diff
@@ -0,0 +1,85 @@
+#' Generate retrospective nowcasts
+#'
+#' This function ingests a list of retrospective reporting triangles and
+#'   generates a list of retrospective reporting squares, or "complete"
+#'   reporting triangles. It uses the specified `n_history_delay`  number of
+#'   observations to estimate the empirical delay for each retrospective
+#'   reporting triangle.
+#'
+#' @param list_of_rts List of reporting triangle matrices, in order
```

Comment:
what is the name of this output in estimate_triangles. Seems like this should be similar

Reply:
Will go through and make sure the output names match the input names for the strings of functions